server.js

```diff
@@ -12,37 +15,33 @@ app.get('/', (request, response) => {
   response.send('TasteRX API');
 });
 
-app.listen(app.get('port'), () => {
-  console.log(`${app.locals.title} is running on http://localhost:${app.get('port')}.`);
-});
-
-app.locals.counter = 1;
-app.locals.prescriptions = [];
-
-app.post("/prescriptions", (request, response) => {
+app.post("/prescriptions", async (request, response) => {
   const { message, showID } = request.body
   if(!message || !showID) {
-    response.status(422).json( { error: "Expected { message: <String>, showID: <Number>}" } )
-  }
-
-  const newPrescription = {
-    id: app.locals.counter,
-    message: message,
-    showID: showID
+    response.status(422).json( { error: "Expected { message: <String>, showID: <Number> }" } )
   }
-  app.locals.counter ++
-
-  app.locals.prescriptions.push(newPrescription)
-  response.status(201).json({ prescription: newPrescription})
+  const newPrescription = await models.TVPrescription.create({ message: message, showID: showID })
+  response.status(201).json({ prescription: newPrescription })
 })
 
-app.get('/prescriptions/:id', (request, response) => {
+app.get('/prescriptions/:id', async (request, response) => {
     const { id } = request.params
-    
-    const requestedPrescription = app.locals.prescriptions.find( (prescription) => {
-      return prescription.id == id;
-    } )
+    const requestedPrescription = await models.TVPrescription.findOne({ where: { id: id } });
     console.log(requestedPrescription)
-    response.status(201).json({data: requestedPrescription});
+    if (requestedPrescription === null) {
+      response.status(422).json( { error: "Not found" } )
+    } else {
+      response.status(201).json({ data: requestedPrescription });
```

Comment:
@nicktu12 thank you for seeing that, I think it needs to be a 200 error.
```suggestion
      response.status(200).json({ data: requestedPrescription });
```